modules/mq-instance/main.tf

```diff
@@ -18,8 +18,11 @@ resource "ibm_resource_instance" "mqcloud_capacity" {
   name              = var.name
   plan              = "reserved-capacity"
   resource_group_id = var.resource_group_id
-  service           = "mqcloud"
-  tags              = var.tags
+  parameters = {
+    "subscriptionId" = var.subscription_id
```

Comment:
according to what has been told by RC team, the format for the parameter key is `subscription_id`